### PR TITLE
Tester tests

### PR DESCRIPTION
**Pull request by krux02**
add tests where that should be rejected by the tester. Not yet integrated in test environment.

### CODE REVIEW

**Comment by Araq** [comment]
Ok. How to run these tests?

**Comment by krux02** [comment]
```
cd testament
./tester --nim:../compiler/nim category shouldfail
```
and then you have to manually look if they all fail properly. Yea it is kind of bad, I see it. But better than nothing.

**Comment by Araq** [comment]
You can write a metatest for testament. I'll give it a shot.

File: testament/tester.nim [hunk 1]
```diff
@@ -505,7 +539,17 @@ proc main() =
       targetsStr = p.val.string
       targets = parseTargets(targetsStr)
     of "nim": compilerPrefix = p.val.string
-    else: quit Usage
+    of "directory":
+      setCurrentDir(p.val.string)
+    of "colors":
+      if p.val.string == "on":
```
Comment (Araq) [inline testament/tester.nim]: You could use ``strutils.parseBool`` here. (Not required, just that you know.) Or a ``case`` statement for strings.

**Pull request merged**